packages/bisect_ppx/bisect_ppx.0.2.2/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+name: "bisect_ppx"
+version: "0.2.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/bisect_ppx"
+dev-repo: "https://github.com/rleonid/bisect_ppx.git"
+bug-reports: "https://github.com/rleonid/bisect_ppx/issues"
+build: [
+  ["sh" "configure"]
+  [make "all"]
+  [make "install"]
```

Comment:
There is no conflict between the `.install` and the `install` field: you can use both if it's easier for you (e.g call `make install` to install `ocamlfind install` and put the installed binary in the `.install` file). Would be interesting to know where you read that there is a conflict so that we can fix the documentation.